assets/translate.js

```diff
@@ -1,3 +1,16 @@
+var alreadyUpdating = false,
+	enableInputs = function() {
+		alreadyUpdating = true;
```

Comment:
Is there a need for this variable? The loop below should be synchronous.

Reply:
Enabling a control counts as an update, resulting in preview being reloaded for every input being enabled.

Reply:
So, re-reading the ticket and talking to Niharika, it sounds like the inputs should just always start out disabled -- which means we should probably just get them from the php-widgets already disabled, and enable them when the selection of the 'to' language changes.